OneSignalSDK/gradle.properties

```diff
@@ -22,10 +22,10 @@
 #    org.robolectric.res.ResourceTableFactory.parseResourceFiles
 # Remove when creating an .aar build.
 #android.enableAapt2=false
-
-android.databinding.incremental = false
-kapt.incremental.apt = false
-android.testConfig.useRelativePath = false
+#removed this setting because incompatibility with the newest gradle version
```

Comment:
Should we remove the commented lines then?

Reply:
This is incompatible with the gradle 8.x.x that we are using in MABS10

Reply:
I see. Should we remove the lines instead of leaving them commented in the repo?

Reply:
Yes we can